packages/lit-dev-tests/src/playwright/util.ts

```diff
@@ -23,11 +23,42 @@ export async function waitForPlaygroundPreviewToLoad(page: Page) {
     });
     return true;
   }, iframe);
-  // Wait for the loading indicator to stop for screenshots.
-  await page.waitForSelector(
-    'playground-preview [part="preview-loading-indicator"][aria-hidden="true"]'
-  );
-  // There is a fade-out transition on the playground loading bar that makes
-  // screenshots flaky. Wait for the loading bar to have animated out.
-  await page.waitForTimeout(250);
+  // Hide the animated loading indicator.
+  await page.evaluate((el) => {
+    el.style.visibility = 'hidden';
+  }, await page.waitForSelector('playground-preview [part="preview-loading-indicator"]'));
```

Comment:
Really love the constant improvements.
Is this removing the subtle animation dependency by literally forcing a `hidden` style onto the loading bar part? I really like that!

Reply:
Yep! I was fighting with it still sometimes rendering mid-screenshot, and realized it would be simplest to just not show it at all.